aep/general/0300/aep.md.j2

```diff
@@ -0,0 +1,98 @@
+# AEP Editions
+
+This AEP describes how the AEP specification is versioned.
+
+## Edition based versioning
+
+AEPs will be published in editions, similar but not identical to specifications
+such as those for the C++ or Rust programming language:
+
+- The specification will not change for a non-preview edition after it is
+  published.
+- The naming of the edition will be based on the year in which it is published
+  (e.g. aep-2025).
+- Upon publishing a non-preview edition, the current prevision will be removed,
+  and the next preview edition will be created (e.g. aep-2027-preview). This
+  preview edition will accumulate changes until the next edition is published.
+- AEP editions **MAY** have patch version updates which adhere to semantic
+  versioning (e.g. aep-2025.25).
+  - The patch version will only contain fixes to typos and clarification on
+    existing guidance.
+- Editions will be published every 2 years.
+
+## Breaking changes between editions
+
+Although not strictly related to versioning, the AEP specification will
+generally strive to minimize breaking changes, even across editions.
+
+## First-party clients and tooling
+
+_NOTE_: This proposal only applies to AEP first-party clients. Third party open
+source projects or organizations are governed outside of this project, and may
+have their own guarantees.
+
+### Client versioning
+
+- Clients _SHOULD_ adhere to [Semantic Versioning 2.0.0](https://semver.org/)
+
+### Client and edition compatibilty
+
+The newest major version of clients and tools **MUST** be compatible with, at
+minimum, the 3 latest non-preview AEP editions.
+
+The following guidance applies to the most recent major versions of clients:
+
+- Clients may provide different support guarantees for older major versions.
+- Each major version will state what AEP editions are supported by those
+  clients.
+- This guidance does not apply to accepting PRs on older major versions: those
+  may be accepted for any maintained branch of the project, at the discretion
+  of the client maintainers.
+- Clients and tools may support features in preview editions, but support for
+  preview edition features in all clients and tools are not guaranteed.
+
+## Examples of AEP specification changes
+
+The following are examples of specification changes:
+
+- renaming a field (e.g. name to path syntax)
```

Comment:
```suggestion
- renaming a field (e.g. `name` to `path`)
```
I'm not sure if this is a clarification or if you meant to refer to something else, given "syntax".